common/reconciliation/invariant/concreteExecutionExists.go

```diff
@@ -34,15 +35,17 @@ import (
 type (
 	concreteExecutionExists struct {
 		pr persistence.Retryer
+		dc cache.DomainCache
 	}
 )
 
 // NewConcreteExecutionExists returns a new invariant for checking concrete execution
 func NewConcreteExecutionExists(
-	pr persistence.Retryer,
+	pr persistence.Retryer, dc cache.DomainCache,
 ) Invariant {
 	return &concreteExecutionExists{
 		pr: pr,
+		dc: dc,
```

Comment:
domainCache is added but not used.
The same thing applies to other files in this package.